lib/services/assets-generation/assets-generation-service.ts

```diff
@@ -12,21 +12,23 @@ export class AssetsGenerationService implements IAssetsGenerationService {
 	@exported("assetsGenerationService")
 	public async generateIcons(resourceGenerationData: IResourceGenerationData): Promise<void> {
 		this.$logger.info("Generating icons ...");
-		await this.generateImagesForDefinitions(resourceGenerationData.imagePath, resourceGenerationData.resourcesPath, Icons);
+		await this.generateImagesForDefinitions(resourceGenerationData.imagePath, resourceGenerationData.resourcesPath, Icons, resourceGenerationData.platform);
```

Comment:
Can we introduce an interface and pass object as param?

Reply:
As it is a private method and I have some concerns about polluting the .d.fs only for this method I will leave it as it is.